prisma.config.ts

```
@@ -0,0 +1,16 @@
+
+// This file was generated by Prisma and assumes you have installed the following:
+// npm install --save-dev prisma dotenv
+import "dotenv/config";
+import { defineConfig, env } from "prisma/config";
+
+export default defineConfig({
+  schema: "prisma/schema.prisma",
+  migrations: {
+    path: "prisma/migrations",
+  },
+  engine: "classic",
```

ESLint

[error] Replace `"classic"` with `'classic'`

(prettier/prettier)